python/django/security/csrf_exempt.yaml

```diff
@@ -0,0 +1,12 @@
+rules:
+  - id: django-csrf_exempt
+    pattern: |
+      from django.views.decorators.csrf import csrf_exempt
+      ...
+
+      @csrf_exempt
```

Comment:
Would import resolution work here, too?

e.g., 
`@django.views.decorators.csrf.csrf_exempt`

Reply:
I tried that in the rule but let me make this an or case